include/EclipseMonitor/Trie/BranchNode.hpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 Tuan Tran
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+#pragma once
+
+#include "EclipseMonitor/EthKeccak256.hpp"
+#include "EclipseMonitor/Trie/Nibbles.hpp"
+#include "EclipseMonitor/Trie/TrieNode.hpp"
+#include "SimpleObjects/Internal/make_unique.hpp"
```

Comment:
For external projects, use absolute path with angle brackets, so this will be `#include <SimpleObjects/Internal/make_unique.hpp>`.

Also, place this before the include statements referencing the header files within the same project, since the order of include statements should be
- the corresponding header file (if this file is a `*.cpp` file)
- standard C headers (e.g., `#include <cmath>`)
- standard C++ headers (e.g., `#include <vector>`)
- external projects
- internal headers

Also, try to keep them in alphabetical order when remember (it's ok if you forget sometimes)